code/datums/elements/blood_walk.dm

```diff
@@ -0,0 +1,45 @@
+///Blood walk, a bespoke element that causes you to make blood wherever you walk.
+/datum/element/blood_walk
+	element_flags = ELEMENT_BESPOKE
```

Comment:
```suggestion
	element_flags = ELEMENT_BESPOKE|ELEMENT_DETACH
```

Reply:
I was doing removeelement to avoid using this but yea ill use it instead

Reply:
i mean im sure you could do that instead, but i figured this was probably better practice so its all contained within the element

Reply:
it very much likely is better, since bubblegum would probably harddel if admins VV edit spawn_blood after it was given the element, and not having the check would make it runtime if bubblegum spawned without spawn_blood.